src/Components/Navbar.js

```diff
@@ -43,7 +43,7 @@ function Navbar({ items, start, end }) {
 
         <div style={theme}>v{version}</div>
 
-        <div className="theme--button">
+        <a href="#" className="theme--button">
```

Comment:
Link is not the right semantic element to choose here because it's not leading anywhere. A button would be the most appropriate as it performs an action.

Reply:
Ok I will fix it soon.